aws/validators.go

```diff
@@ -1625,6 +1625,19 @@ func validateIoTTopicRuleElasticSearchEndpoint(v interface{}, k string) (ws []st
 	return
 }
 
+func validateIoTTopicRuleFirehoseSeparator(v interface{}, s string) ([]string, []error) {
+	switch v.(string) {
+	case
+		",",
+		"\t",
+		"\n",
+		"\r\n":
+		return nil, nil
+	}
+
+	return nil, []error{fmt.Errorf("Separator must be one of ',' (comma), '\t' (tab) '\n' (newline) or '\r\n' (Windows newline)")}
```

Comment:
When printing this error, it is returning the whitespace and not the intended characters:

```
aws_iot_topic_rule.rule: Separator must be one of ',' (comma), '	' (tab) '
        ' (newline) or '
        ' (Windows newline)
```

They need to be escaped appropriately. 👍